native/src/components/Categories.tsx

```diff
@@ -82,21 +85,23 @@ const Categories = ({
       path={category.path}
       AfterContent={category.organization && <OrganizationContentInfo organization={category.organization} />}
       Footer={
-        <List
-          items={children}
-          renderItem={({ item: it }) => (
-            <CategoryListItem
-              key={it.path}
-              category={it}
-              subCategories={categories.getChildren(it)}
-              resourceCache={resourceCache}
-              language={language}
-              onItemPress={navigateToCategory}
-            />
-          )}
-          // Fixes VirtualizedLists nesting error
-          scrollEnabled={false}
-        />
+        <>
+          <EmbeddedOffer embeddedOffers={embeddedOffers} languageCode={language} />
```

Comment:
I think the offers should be rendered below the list. Otherwise the categories are only after an possibly very long list.

Reply:
Yes, that's true. But I think design wise the offer belongs to the content above. Maybe we should disallow embedded offers in second level pages...